safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py

```diff
@@ -182,75 +182,66 @@ def action_checklist(self):
         total_fatalities = self.total_fatalities
         total_displaced = self.total_evacuated
         rounded_displaced = format_int(population_rounding(total_displaced))
-        message = m.Message(style_class='container')
-        message.add(m.Heading(tr('Action checklist'), **styles.INFO_STYLE))
-        checklist = m.BulletedList()
+
+        title = 'Action checklist'
+        fields = []
         if total_fatalities:
-            checklist.add(tr(
+            fields.append(tr(
                 'Are there enough victim identification units available '
                 'for %s people?') % (
                     format_int(population_rounding(total_fatalities))))
         if total_displaced:
-            checklist.add(tr(
+            fields.append(tr(
                 'Are there enough shelters and relief items available for '
                 '%s people?') % rounded_displaced)
         if rounded_displaced:
-            checklist.add(tr(
+            fields.append(tr(
                 'If yes, where are they located and how will we '
                 'distribute them?'))
         if total_displaced:
-            checklist.add(tr(
+            fields.append(tr(
                 'If no, where can we obtain additional relief items '
                 'from and how will we transport them?'))
-        message.add(checklist)
-        return message
+
+        return {
+            'title': title,
+            'fields': fields
+        }
 
     def notes(self):
         """Notes and caveats for the IF report.
 
-        :returns: List of dicts containing notes.
-        :rtype: list
+        :returns: Dicts containing notes.
+        :rtype: dict
         """
-        message = m.Message(style_class='container')
-        message.add(
-            m.Heading(tr('Notes and assumptions'), **styles.INFO_STYLE))
-
-        checklist = m.BulletedList()
-
-        checklist.add(tr(
-            'Total population in the analysis area: %s'
-            ) % format_int(population_rounding(self.total_population)))
-
-        checklist.add(tr(
-            '<sup>1</sup>People are displaced if '
-            'they experience and survive a shake level'
-            'of more than 5 on the MMI scale.'))
-
-        checklist.add(tr(
-            'The fatality calculation assumes that '
-            'no fatalities occur for shake levels below 4 '
-            'and fatality counts of less than 50 are '
-            'disregarded.'))
-
+        title = tr('Notes and assumptions')
+        fields = [
+            tr('Total population in the analysis area: %s') %
+            format_int(population_rounding(self.total_population)),
+            tr('<sup>1</sup>People are displaced if they experience and '
```

Comment:
We probably need to get rid of mark up embedded in the text like this as it won't map well to other representations e.g. plain text.